go/chat/bots/commands.go

```diff
@@ -152,8 +151,14 @@ func (b *CachingBotCommandManager) createConv(ctx context.Context, param chat1.A
 			return res, errors.New("missing team name")
 		}
 		topicName := fmt.Sprintf("___keybase_botcommands_team_%s_%v", username, param.Typ)
+		var membersType chat1.ConversationMembersType
+		if _, err := teams.Load(ctx, b.G().GlobalContext, keybase1.LoadTeamArg{Name: *param.TeamName}); err == nil {
```

Comment:
Sorry to keep nitpicking, but I wonder if there is a specific error we could look for here. Like if I get "APINetError" back here it would be a shame to then just plow forward thinking its an imp team